library/Zend/Code/Generator/ClassGenerator.php

```diff
@@ -486,23 +491,6 @@ public function addPropertyFromGenerator(PropertyGenerator $property)
     }
 
     /**
-     * Add a class to "use" classes
-     *
-     * @param  string $use
-     * @param  string|null $useAlias
-     * @return ClassGenerator
-     */
-    public function addUse($use, $useAlias = null)
```

Comment:
Was in the middle of commenting that removing this is a BC break, and suggesting to have it proxy to the `TraitUsageGenerator`... and then saw that you just moved the method to later in the class. :)